huaweicloud/services/acceptance/as/resource_huaweicloud_as_configuration_test.go

```diff
@@ -77,6 +77,79 @@ func TestAccASConfiguration_instance(t *testing.T) {
 	})
 }
 
+func TestAccASConfiguration_NewDiskTypeAndDEH(t *testing.T) {
```

Comment:
How about remove new disk type test cace to `TestAccASConfiguration_sharedBandwidth`.
And remove func name from `TestAccASConfiguration_sharedBandwidth` to `TestAccASConfiguration_bandwidth_new_disk`

Reply:
done